Flowsheet_Class/flowsheet.py

```
@@ -673,60 +705,74 @@ def split_HI_nodes(self, OntoCapeNames=False):
         edges_to_remove = [k for k, v in edge_information_signal.items() if v == ["not_next_unitop"]]
         flowsheet_wo_signals.remove_edges_from(edges_to_remove)
 
+        # First get the names of the HEX we actually have to split:
+        hex_to_split = []
+        for node_name, node_attrs in flowsheet_wo_signals.nodes(data=True):
```

Ruff (0.15.7)

[warning] Loop control variable `node_attrs` not used within loop body

(B007)